app/views/shared/_nav.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= stylesheet_link_tag 'nav' %>
+<header>
+  <nav>
+    <%= link_to "Foods", foods_path %>
+    <%= link_to "Recipes" %>
+    <%= link_to "Public Recipes" %>
+    <%= link_to "Shopping List" %>
+    <%= link_to "Sign Up" %>
+    <%= link_to "Sign In" %>
+    <%= link_to "Sign Out" %>
```

Comment:
Nice navigation implementation! Please use the correct link to navigate to the appropriate page. 😉